### PR TITLE
Performance Enhancements

### PR DESCRIPTION
**Pull request by Postremus**
Profiling was started before clicking on Refresh, and stopped after.

Before my changes:
ss 2015-07-10 at 11 42 35

After my changes:
ss 2015-07-10 at 11 38 26

### CODE REVIEW

**Comment by pjf** [comment]
puu.sh seems to be having troubles right now, and I'd love to see the performance graphs! :)

**Comment by Postremus** [comment]
Sorry for them not being available. uploaded them to github.

File: GUI/MainRepo.cs [hunk 1]
```diff
@@ -72,7 +72,6 @@ private void UpdateRepo(object sender, DoWorkEventArgs e)
         private void PostUpdateRepo(object sender, RunWorkerCompletedEventArgs e)
         {
             SetDescription("Scanning for manually installed mods");
```
Comment (RichardLake) [inline GUI/MainRepo.cs]: Shouldn't this line also be removed?

Comment (Postremus) [inline GUI/MainRepo.cs]: Why?

Comment (Postremus) [inline GUI/MainRepo.cs]: ah, I see the reason :)

**Comment by Postremus** [comment]
rebased the pr.

**Pull request merged**